src/writeData/components/PluginCreateConfiguration/Footer.tsx

```diff
@@ -86,6 +92,8 @@ const FooterComponent: FC<Props> = props => {
 
   const handleSaveAndTest = () => {
     onSaveTelegrafConfig()
+    setBucketInfo('', '', '')
+    setLocationOnDismiss(`/orgs/${orgID}/load-data/telegrafs`)
```

Comment:
The act of clicking "Save and Test" means the user has completed the creation of a Telegraf configuration. (The testing part is optional and the user can dismiss that step without consequence.)

After creating a Telegraf configuration, we always drop the user off in Load Data > Telegrafs. We also forget the last selected bucket. Additional user actions will re-select a bucket, as appropriate.